clients/js/cmds/sui.ts

```diff
@@ -16,6 +17,31 @@ function assertNetwork(n: string): asserts n is Network {
   }
 }
 
+/*
+  Loop through a list of Sui objects and look for the DeployerCapability that should
+  have been granted upon publication of the package with package_id
+
+  The objects is in the format returned by a json-rpc call "provider.getObjectsOwnedByAddress"
+*/
+function findDeployerCapability(package_id: string, module_name: string, objects: Array<any>): string {
+      let deployer;
+      for (var obj of objects){
+        let t = obj.type;
+        // do a basic check and see if the type is as expected (has a normal package, module, type name separated by "::")
+        var count = [...t].filter(x => x === ":").length;
```

Comment:
we have a method `isValidAptosType` in the SDK that I think would a better way to check if this is a valid fully qualified type